lib/utils/get-note.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import filterNotes from './filter-notes';
+
+export default function getNote( state ) {
+	if ( state.note ) {
+		return state.note;
+	}
+	const filteredNotes = filterNotes( state );
+	const noteIndex = Math.max( state.previousIndex, 0 );
+	return filteredNotes[ noteIndex ];
+}
```

Comment:
I'm a little rusty on this, but IIRC it's _all_ of them.
Like:
- if there's a selected note → return the selected note
- else if there's a note list index stored → return the note corresponding to that index (if I remember this right, this is needed when you delete a note, to send you back to a previously opened note)
- else → return the first note

Maybe `getCurrentNote` might be more apt, but it wouldn't really clear the meaning of this function.
Probably the best approach would be a good old comment.